src/lib/REST/Output/ValueObjectVisitor/ContentTree/NodeExtendedInfo.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace Ibexa\AdminUi\REST\Output\ValueObjectVisitor\ContentTree;
+
+use Ibexa\Contracts\Rest\Output\Generator;
+use Ibexa\Contracts\Rest\Output\ValueObjectVisitor;
+use Ibexa\Contracts\Rest\Output\Visitor;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * @phpstan-import-type TPermissionRestrictions from \Ibexa\AdminUi\REST\Value\ContentTree\NodeExtendedInfo
+ */
+final class NodeExtendedInfo extends ValueObjectVisitor
```

Comment:
Naming: `NodeExtendedInfoVisitor`. The current name suggests it's a value and can be easily confused with the actual REST value object class `NodeExtendedInfo`. I know this can be inferred from the namespace, but this pattern is not practical in many cases as it makes navigating through the code more difficult.